doc/en/how-to/cache.rst

```diff
@@ -2,8 +2,8 @@
 .. _cache:
 
 
-Cache: working with cross-testrun state
-=======================================
+How to re-run failed tests with the cache plugin
```

Comment:
The `--last-failed` functionality is certainly what the cache plugin is most used for, but as the previous title says, it is a more general system for keeping state across test runs/invocations. I don't know how long the titles can be, but it would be ~nice if that aspect can be preserved.